frontend/packages/core/src/Feedback/error/details.tsx

```diff
@@ -0,0 +1,186 @@
+    state: "expanded",
+  });
+
+  React.useEffect(() => {
+    setDetailsOpen(false);
+  }, [error]);
+
+  const hasWrappedErrorDetails =
+    error.details.filter(detail => isClutchErrorDetails(detail)).length > 0;
+
+  const summaryIconStyle = { marginRight: "8px" };
+
+  return (
+    <>
+      <ErrorDetailDivider />
+      <Accordion elevation={0} expanded={expanded}>
+        <AccordionSummary
+          aria-controls="panel1a-content"
+          data-expanded={expanded}
+          onClick={() => setExpanded(!expanded)}
+        >
+          {!expanded ? (
+            <>
+              <ChevronRightIcon style={summaryIconStyle} /> Show more
+            </>
+          ) : (
+            <>
+              <KeyboardArrowDownIcon style={summaryIconStyle} /> Show less
+            </>
+          )}
+        </AccordionSummary>
+        <AccordionDetails>
+          <ErrorDetailContainer>
+            {hasWrappedErrorDetails && (
+              <div>
+                <ErrorDetailText style={{ color: "#0D1030" }}>
+                  The following errors were encountered:
+                </ErrorDetailText>
+                <ul>
+                  {error.details.map(detail => {
+                    // Only render Clutch Error wrapped details errors here
+                    if (isClutchErrorDetails(detail)) {
+                      const renderMax = 2;
+                      const renderItems = detail.wrapped.slice(0, renderMax);
+                      const remainingItems = detail.wrapped.length - renderMax;
+                      return (
+                        <>
+                          {renderItems.map((wrapped, idx) => {
+                            const color =
+                              grpcCodeToHttpCode(wrapped.code) >= 500 ? "#DB3615" : "#daca37";
+                            return (
+                              // eslint-disable-next-line react/no-array-index-key
+                              <ListItem key={`${idx}-${wrapped.message}`}>
+                                <ErrorDetailText style={{ fontWeight: 500 }}>
+                                  <span style={{ color }}>
+                                    {grpcCodeToText(wrapped.code)}&nbsp;
+                                  </span>
+                                  {wrapped.message}
+                                </ErrorDetailText>
+                              </ListItem>
+                            );
+                          })}
+                          {remainingItems > 0 && (
+                            <ErrorDetailText>and {remainingItems} more...</ErrorDetailText>
+                          )}
+                        </>
+                      );
+                    }
+                    return null;
+                  })}
+                </ul>
+              </div>
+            )}
+            <Grid container justify="flex-end">
+              <DialogButton onClick={() => setDetailsOpen(true)}>More Details</DialogButton>
```

Comment:
this is all uppercase which i don't think we have anywhere else in clutch for buttons

Reply:
also it should be left-aligned

Reply:
The design has all uppercase and right-aligned

Reply:
ah ok fine for now, i can play with some changes after merge that are closer to one of the other designs